File: temp_conv.py
class TemperatureConverter:
    value = 0

    def __init__(self):
        self.value = 0

    def toCelcius(self):
        return (self.value - 32) * 5/9

    def toFahrenheit(self):
        return self.value * 9/5 + 32;

tc = TemperatureConverter(32)
println tc.toFahrenheit()

File: temp_conv.rb
# Class

class TemperatureConverter
  def initialize(temp)
    @value = temp
  end

  def toCelcius
    return (self.value - 32) * 5/9
  end

  def toFahrenheit
    return self.value * 9/5 + 32;
  end

tc = TemperatureConverter.new(212)
print tc.toCelcius
